patches/server/1044-Add-skipping-world-symlink-scan.patch

```diff
@@ -0,0 +1,24 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: mja00 <me@mja00.dev>
+Date: Mon, 12 Aug 2024 06:27:15 -0400
+Subject: [PATCH] Add skipping world symlink scan
+
+In worlds that are extremely large (greater than 1TB), it can take an insanely long time to walk the entire world for symlinks.
+This patch adds a system property to disable the symlink scan, which can be used to speed up world loading.
+A good example of this issue can be found in the Discord (https://discord.com/channels/289587909051416579/289587909051416579/1271527497674129428)
```

Comment:
The relevant part is `In worlds that are extremely large (greater than 1TB), it can take an insanely long time to walk the entire world for symlinks.`
But yea, I guess it can be removed, not really much benefit in that link.

Reply:
Fair. Was on the fence about adding the link anyways.